tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -126,7 +126,7 @@ var _ = Describe("odo devfile push command tests", func() {
 				[]string{"stat", "/test/server.js"},
 				func(cmdOp string, err error) bool {
 					statErr = err
```

Comment:
You can remove ```statErr = err```

Reply:
We are also verifying that err should not have occured here - https://github.com/openshift/odo/pull/3645/files#diff-c127bd883f89dd3a7abff6183ad9921dR157 . So keeping this in the test